src/components/TrapFocus/TrapFocus.tsx

```diff
@@ -14,14 +14,36 @@ export interface Props {
   children?: React.ReactNode;
 }
 
-export default class TrapFocus extends React.PureComponent<Props, never> {
-  private focusTrapWrapper: HTMLElement | null = null;
+export interface State {
+  shouldFocusSelf: boolean | undefined;
```

Comment:
`shouldFocusSelf?: boolean`

Reply:
It needs to initialize as undefined (see below)